docs/miscellaneous.md

```diff
@@ -7,6 +7,7 @@
 - [Launch Logged-In Window](#launch-logged-in-window)
 - [User account requirements](#user-account-requirements)
 - [List of wrapper repos & friends](#list-of-wrapper-repos--friends)
+- [What to name your branch](#naming-branches)
```

Comment:
I coudn't get the anchor link to work - I think the anchor needs to be something like #what-to-name-your-branch to work

Reply:
I will take another look.